src/features/farming/animals/components/MutantAnimalModal.tsx

```diff
@@ -85,24 +21,52 @@ interface Props {
 
 export const MutantAnimalModal = ({ mutant, show, onContinue }: Props) => {
   const { t } = useAppTranslation();
+  const boost = COLLECTIBLE_BUFF_LABELS[mutant];
+  const isPermanentMutant = (name: string): boolean => {
+    const permanentMutants = ["Speed Chicken", "Rich Chicken", "Fat Chicken"];
+    return permanentMutants.includes(name);
+  };
   return (
-    <Modal show={show}>
+    <Modal show={show} dialogClassName="max-w-[480px]">
       <Panel>
-        <div className="p-2">
-          <h1 className="text-lg text-center">
+        <div className="flex flex-wrap justify-between m-1 ml-2 mb-1">
+          <Label type="warning" icon={chest}>
+            {t("reward")}
+          </Label>
+          {isPermanentMutant(mutant) ? (
+            <Label className="mr-0.5" type="vibrant" secondaryIcon={Crown}>
+              {t("permanent")}
+            </Label>
+          ) : (
+            <Label type="info" secondaryIcon={SUNNYSIDE.icons.stopwatch}>
+              {t("seasonal")}
+            </Label>
+          )}
```

Comment:
These labels are a little bit redundant since only seasonal ones can drop now

Reply:
I think you can leave the seasonal one because it reinforces that all mutants are now seasonal but yeah the permanent one can be removed.

Reply:
Thank you for the reviews 🙏. I was working on making it clear to players that mutant animals only drop during the season they're introduced.
![image](https://github.com/user-attachments/assets/cfbc1f41-5408-4e54-afb8-7c0a6f6564af)


But if the season's name is long, it doesn't look good, and  `Seasonal` text shows better in the modal. with respect, I will change it to just show `Seasonal` text.
![image](https://github.com/user-attachments/assets/fd246057-e3e8-4a4c-a136-f182c5190b8e)